### ISSUE TITLE
v7.9 3d-tiles cannot require in this environment

### ISSUE THREAD

**Issue by hardreddata** [TerriaJS/terriajs#3836]
We have an example which works in a cloned TerriaMap from 21 October but not the new version.

The catalogue item is basically name, type, assetId

From the Master CI

```
DeveloperError: Cannot `require` in this environment.
DeveloperError@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:149600:19
requireWasmWrapper@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:183930:19
getWebAssemblyLoaderConfig@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:183955:29
./node_modules/terriajs-cesium/Source/Core/TaskProcessor.js/TaskProcessor.prototype.initWebAssemblyModule@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:184074:35
./node_modules/terriajs-cesium/Source/Scene/DracoLoader.js/DracoLoader._getDecoderTaskProcessor@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:232979:23
./node_modules/terriajs-cesium/Source/Scene/DracoLoader.js/DracoLoader.decodeModel@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:233182:48
./node_modules/terriajs-cesium/Source/Scene/Model.js/Model.prototype.update@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:250724:78
./node_modules/terriajs-cesium/Source/Scene/Batched3DModel3DTileContent.js/Batched3DModel3DTileContent.prototype.update@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:215231:21
./node_modules/terriajs-cesium/Source/Scene/Cesium3DTile.js/Cesium3DTile.prototype.process@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:221558:23
processTiles@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:226713:22
./node_modules/terriajs-cesium/Source/Scene/Cesium3DTileset.js/Cesium3DTileset.prototype.prePassesUpdate@http://ci.terria.io/master/build/vendors~index.TerriaMap.js:226573:21
./node_modules/terriajs-cesium/Source/Scene/PrimitiveCollection.js/PrimitiveCollection.prototype.prePassesUpdate@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:48907:27
prePassesUpdate@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:54090:20
tryAndCatchError@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:54183:30
./node_modules/terriajs-cesium/Source/Scene/Scene.js/Scene.prototype.render@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:54237:25
./node_modules/terriajs-cesium/Source/Widgets/CesiumWidget/CesiumWidget.js/CesiumWidget.prototype.render@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:68462:25
render@http://ci.terria.io/master/build/vendors~3D.TerriaMap.js:67802:32
```

Thanks

**Comment by kring** [comment]
Looks like another victim of Cesium's move to ES6. Can you tell us more about what you're loading when this happens?

**Comment by hardreddata** [comment]
Thanks Kevin,

It is a 60MB textured mesh produced from a drone, uploaded to cesium
through the WebODM plugin. It probably began life as an .obj.

I will "run the bananas"
https://community.opendronemap.org/t/banana-dataset/2516 to try and
reproduce and then share the asset via email.

**Comment by hardreddata** [comment]
I have emailed the banana 3d-tiles example which reproduces the error.

**Comment by kring** [comment]
Thanks for the report @RussellGrew. This is fixed in TerriaJS v7.10.0.
